### DISCUSSION TITLE
Software Updates always show a reboot is needed

### DISCUSSION THREAD

**Discussion by davidmacdonald80** [cockpit-project/cockpit#20414] [Q&A]
This is only happening on one of my computers.  It's on almalinux 9.3 kernel 5.14.0-362.24.1.el9_3.x86_64. Cockpit version 300.4. I don't see any errors related to cockpit anywhere. Is there a cache somewhere that I can clear, or are there any other ideas how to get it to quit saying the computer needs a reboot? 
It doesn't matter how many times i reboot or if there are any packages available to update.
I have other computers on the same os/kernel/version and this isn't happening.

**Comment by martinpitt** [comment]
That sounds like a [tracer issue](https://github.com/FrostyX/tracer). What does `sudo tracer` say on that machine?

**Comment by davidmacdonald80** [comment]
Thanks! that pointed me in the right direction.  It's not specifically related to cockpit. I imagine when I get the root issue resolve, this will too.
